src/daemon/abrtd.c

```diff
@@ -192,6 +192,7 @@ static void queue_post_create_process(struct abrt_server_proc *proc)
                 abrt_g_settings_dump_location, abrt_g_settings_nMaxCrashReportsSize,
                 kind, worst_dir);
 
+        free(worst_dir);
         g_autofree char *deleted = g_build_filename(abrt_g_settings_dump_location ? abrt_g_settings_dump_location : "", worst_dir, NULL);
         worst_dir = NULL;
```

Comment:
This doesn't look right. `worst_dir` is used in the call to `g_build_filename()`. I suggest `g_clear_pointer()` afterwards to perform both the free and nullifying at the same time.

Reply:
Of course! I looked at a wrong line of the diff with the original change and overlooked the arg to `g_build_filename`. Thanks!